lib/puma/dsl.rb

```diff
@@ -561,7 +561,7 @@ def directory(dir)
     end
 
     # Preload the application before starting the workers; this conflicts with
-    # phased restart feature. This is off by default.
+    # phased restart feature. On by default if your app uses more than 1 worker.
```

Comment:
...via `WEB_CONCURRENCY` 😔

Perhaps we should do https://github.com/puma/puma/issues/2483 right away instead